.travis.yml

```diff
@@ -1,38 +1,42 @@
 sudo: required
-language: java
+language: python
```

Comment:
Hm, does that work? java is typically required for the `nextflow ....` part.

Reply:
mmm I think because the line below, I really didn't change this. It is the same here:https://github.com/nf-core/rnaseq/blob/master/.travis.yml

Reply:
Okay, double checked - sometimes people use java and sometimes python. Doesn't matter as long as the tests run fine which they do 👍